src/h/opengl.h

```diff
@@ -555,8 +555,7 @@ struct fontsymbol {
 //#define CWIDTH 0.25
 //#define CNEAR 0.25
 //#define CFAR 50000.0
-//#define CHEIGHT(w) \
-//  (CWIDTH*((double)w->window->height)/((double)w->window->width))
+//#define CHEIGHT(w) (CWIDTH*((double)w->window->height)/((double)w->window->width))
```

Comment:
Are you referring to this line @Don-Ward ?

Reply:
Yes

Reply:
You added `//` at the front of the second line. My compiler was still complaining about "multi line comment".  I move it all to be one line. So this is a new change

Reply:
OK